lib/gitlab/client/runners.rb

```diff
@@ -0,0 +1,124 @@
+class Gitlab::Client
+  # Defines methods related to runners.
+  # @see https://github.com/gitlabhq/gitlabhq/blob/master/doc/api/runners.md
+  module Runners
+
+    # Get a list of specific runners available to the user.
+    # Full runner params documentation: https://github.com/gitlabhq/gitlabhq/blob/master/doc/api/runners.md
+    #
+    # @example
+    #   Gitlab.runners
+    #   Gitlab.runners(:active)
+    #   Gitlab.runners(:paused)
+    #
+    # @param  [Hash] options A customizable set of options.
+    # @option options [String] :scope The scope of specific runners to show, one of: active, paused, online; showing all runners if none provided
+    # @return [Array<Gitlab::ObjectifiedHash>]
+    def runners(options = {})
+      if options[:scope]
+        get("/runners", query: options)
+      else
+        get("/runners")
+      end
```

Comment:
I was following the convention from https://github.com/NARKOZ/gitlab/blob/master/lib/gitlab/client/projects.rb#L15

```
    def projects(options={})
      if options[:scope]
        get("/projects/#{options[:scope]}", query: options)
      else
        get("/projects", query: options)
      end
    end
```
but I guess not.